src/locale/locales/de/messages.po

```diff
@@ -663,11 +649,11 @@ msgstr "Beschreibung"
 
 #: src/view/com/auth/create/Step1.tsx:96
 msgid "Dev Server"
-msgstr "Entwicklungs-Server"
+msgstr "Entwicklungsserver"
 
 #: src/view/screens/Settings.tsx:637
 msgid "Developer Tools"
-msgstr "Werkzeuge für Entwickler"
+msgstr "Entwickler-Tools"
 
 #: src/view/com/composer/Composer.tsx:210
 msgid "Did you want to say anything?"
```

Comment:
"Möchtest du nichts schreiben?"

Reply:
`"Werkzeuge für Entwickler"` is no bad German